docs/running-clusterfuzzlite/gitlab.md

```diff
@@ -226,12 +252,17 @@ For continuous builds, you need to use a [cache](https://docs.gitlab.com/ee/ci/c
 The cache directory needs to defined as `CFL_CACHE_DIR` to be used by ClusterFuzzLite.
 If it is not defined, the default value is `cache`.
 You should ensure that the runners share the access to the cache.
+As such, this filestore is not available with sharer runners on Gitlab.com and you
```

Comment:
Fixed the typo

As I see the code in cifuzz, only `upload_build` (and download) imply the use of a cache because `GitFilestore` does not handle builds (only coverage and corpus)

IIRC, there was some discussion where I pointer that artifacts cannot be used (because they cannot be read from another gitlab pipeline).

Maybe I can try to revisit this, maybe try https://docs.gitlab.com/ee/api/projects.html#upload-a-file

Reply:
But https://gitlab.com/gitlab-org/gitlab/-/issues/25838 is still open...

Reply:
Surprisingly, that seems to work on gitlab.com shared runner 

> Checking cache for clusterfuzz-lite-non_protected...
Downloading cache.zip from https://storage.googleapis.com/gitlab-com-runners-cache/project/39545533/clusterfuzz-lite-non_protected

Gitlab Runners seem to share the cache on some google storage even if that is not a hard guarantee